.github/workflows/gradle.yml

```diff
@@ -37,5 +35,43 @@ jobs:
       run: sudo apt-get install omlib-modelica-3.0
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
-    - name: Test with Gradle
-      run: ./gradlew test -i
+    - name: Build with Gradle
+      run: ./gradlew build -i
+  release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out ${{ github.ref }}
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build Jars with Gradle
+      run: ./gradlew shadowClientJar shadowServerJar
+    - name: Add version from build.gradle to output
+      id: version
+      run: echo "::set-output name=version::$(./gradlew getVersion -q)"
+    - name: Print Version
+      run: echo ${{ steps.version.outputs.version }}
+    - name: Release
+      uses: softprops/action-gh-release@v1
+      with:
+        files: |
+          ./build/libs/thm.mope.lsp-${{ steps.version.outputs.version }}-Server.jar
+          ./build/libs/thm.mope.lsp-${{ steps.version.outputs.version }}-ConsoleClient.jar
+        token: ${{ secrets.GITHUB_TOKEN }} #Provided by GithubActions
+        target_commitish: ${{github.sha}} #Get latest commit on this branch
+        draft: false
+        prerelease: true
+        tag_name: v${{ steps.version.outputs.version }}
+        name: Release v${{ steps.version.outputs.version }}
+        fail_on_unmatched_files: true
+        body: |
+          Changes in this Release
+          - separating build and release
```

Comment:
I am sure you are aware of that, but this should be replaced by a dynamic script generating the release body from the change log or any other source on the file system. Hard-coding the release notes in the Gradle script is a little clunky.